src/components/export/Export/Export.test.js

```diff
@@ -88,20 +89,19 @@ initialState.openedFacilityTabs.export = [
 ];
 let store = configureStore(initialState);
 //testing redux connected component to mimic props passed as argument
-const componentRenderer = () => {
-  const props = {
-    orisCode: 51,
-    selectedConfig: initialState.openedFacilityTabs.export[0].selectedConfig,
-    title : "Barry (1, 2, CS0AAN)",
-  };
-  return render(
-    <Provider store={store}>
-      <Export {...props} />
-    </Provider>
-  );
+
+const props = {
+  orisCode: 51,
+  selectedConfig: initialState.openedFacilityTabs.export[0].selectedConfig,
+  title: "Barry (1, 2, CS0AAN)",
 };
 
-test("testing Export component renders with no errors", async () => {
-  let { container } = await waitForElement(() => componentRenderer());
-  expect(container).toBeDefined();
-});
+test('renders Export', async () => {
+  await act(() => {
+    render(
```

Comment:
render function is already wrapped with act and a provider if you pass in store. you can just call it this way
 await render(<Export {...props}/>, store)